cpp/change-notes/2020-02-04-unsigned-difference-expression-compared-zero.md

```diff
@@ -0,0 +1,2 @@
+lgtm
```

Comment:
The other change notes all seem to be `lgtm,codescanning`.
No `codescanning` for this?

Reply:
No. As far as I know, code scanning only runs queries with @precision high, and this one is currently at medium. We hope that this query can be promoted to high precision in the future, but we want to try it out on LGTM first.

Reply:
That's true by default, but there's alternate suites that are recommended to users that want to run more queries. We might want a separate note for code scanning that mentions the `security-extended` and `security-and-quality` suites.